cmd/vela-git/main.go

```diff
@@ -124,6 +124,15 @@ func main() {
 			Name:     "repo.remote",
 			Usage:    "the remote (clone URL) for the repo being cloned",
 		},
+		&cli.StringFlag{
+			EnvVars: []string{
+				"PARAMETER_PULL_REQUEST_TARGET",
+				"GIT_PULL_REQUEST_TARGET",
+				"VELA_PULL_REQUEST_TARGET",
+			},
+			Name:  "repo.pr_target",
+			Usage: "the pull request target branch",
+		},
```

Comment:
This additional parameter looks good 👍 

However, based off my above comment, I wonder if we need another new parameter to make this behavior opt-in?

Thinking perhaps it may look like:

```go
		&cli.BoolFlag{
			EnvVars: []string{"PARAMETER_FETCH_TARGET", "GIT_FETCH_TARGET"},
			Name:  "repo.fetch_target",
			Usage: "enables fetching the pull request target branch",
		},
```